SparkPost/2web2ui
# SA-1993: Industry Benchmark Modal

### What Changed

- Added industry benchmark modal
- Added data to reportBuilderContext
- Added constants for the modal

### How To Test

- Go to /signals/analytics
- Enable deliverability product
- Add inbox folder rate as a metric
- There should be a industry benchmark button on the top-right of the charts
- Click it and a modal should appear.

### To Do

- [ ] Integrate with SA-1997
